python/cudf/dataframe/dataframe.py

```diff
@@ -284,6 +284,14 @@ def __len__(self):
         """
         return self._size
 
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+        if method == '__call__':
+            if 'sqrt' in str(ufunc):
```

Comment:
This feels super weird and brittle, any way we can have a more concrete way to compare the ufunc here?